posix.mak

```diff
@@ -35,17 +36,22 @@ endif
 
 # Set PHOBOS name and full path
 ifeq (,$(findstring win,$(OS)))
-	PHOBOS = $(PHOBOS_PATH)/generated/$(OS)/release/$(MODEL)/libphobos2.a
-	PHOBOSSO = $(PHOBOS_PATH)/generated/$(OS)/release/$(MODEL)/libphobos2.so
+	PHOBOS = $(PHOBOS_PATH)/generated/$(OS)/$(BUILD)/$(MODEL)/libphobos2.a
+	PHOBOSSO = $(PHOBOS_PATH)/generated/$(OS)/$(BUILD)/$(MODEL)/libphobos2.so
 endif
 
 # default to warnings and deprecations as errors, override via e.g. make -f posix.mak WARNINGS=-wi
 WARNINGS = -w -de
 # default include/link paths, override by setting DFLAGS (e.g. make -f posix.mak DFLAGS=-I/foo)
 DFLAGS = -I$(DRUNTIME_PATH)/import -I$(PHOBOS_PATH) \
-		 -L-L$(PHOBOS_PATH)/generated/$(OS)/release/$(MODEL) $(MODEL_FLAG) -fPIC
+		-L-L$(PHOBOS_PATH)/generated/$(OS)/$(BUILD)/$(MODEL) $(MODEL_FLAG) -fPIC
 DFLAGS += $(WARNINGS)
 
+# dragonflybsd: curl is found in /usr/local/lib
+ifeq (dragonflybsd,$(OS))
```

Comment:
That's where it is on FreeBSD as well - in fact, all the BSDs, unless one of them built it into the base system. And if I add this DFLAGS line to my local posix.mak, then the build succeeds, which it never does for me if I don't comment out the tools that require curl (I've always wondered why it worked on the autotester given that it always fails for me).

Reply:
The auto-tester doesn't build the tools repo.

Reply:
Well, that still doesn't explain why anyone doing the release doesn't get build failures when building the tools repo for FreeBSD. Also, if tools isn't built by the autotester, then it really should be built by one of the other CI tools.

Reply:
> Well, that still doesn't explain why anyone doing the release doesn't get build failures when building the tools repo for FreeBSD. 

Again, those tools aren't build:

https://github.com/dlang/installer/blob/master/create_dmd_release/create_dmd_release.d#L525

> Also, if tools isn't built by the autotester, then it really should be built by one of the other CI tools.

It is (see below or e.g. Jenkins does build tools on every PR), but FreeBSD CIs are hard to come by...

Reply:
Well, if they aren't even built for the release, then that would explain why it works for someone else and not me - it doesn't.